src/layouts/Settings/AnalyticsSettings.tsx

```diff
@@ -27,9 +33,24 @@ export const AnalyticsSettings = ({
         </SectionCaption>
       </VStack>
       <VStack spacing="1.5rem" align="flex-start" w="50%">
-        <FormControl isDisabled={isError}>
+        <FormControl isDisabled={isError} isInvalid={isInvalid("pixel")}>
           <FormLabel>Facebook Pixel</FormLabel>
-          <Input w="100%" {...register("pixel")} />
+          <Input
+            w="100%"
+            {...register("pixel", {
+              minLength: {
```

Comment:
Could we also add a regex pattern to ensure that this only accepts digits and not just any string?

Reply:
Good catch, changed in https://github.com/isomerpages/isomercms-frontend/pull/1282/commits/e49d24fec203b33140aeb69ed31e605cbe8e3637